readme.md

```diff
@@ -53,6 +53,8 @@ pokemon.getName(100);
 //=> 'Voltorb'
 pokemon.getName(100, 'de');
 //=> 'Voltobal'
+pokemon.getName(100, 'fr');
```

Comment:
We don't need more language examples. It's already clear that you can pass a language code.